meta-lmp-base/recipes-sota/lmp-device-register/lmp-device-register_git.bb

```diff
@@ -4,7 +4,7 @@ LIC_FILES_CHKSUM = "file://COPYING.MIT;md5=838c366f69b72c5df05c96dff79b35f2"
 
 DEPENDS = "boost curl glib-2.0"
 
-SRCREV = "dd8fe0a9741ef05951b3fb213fb64f5896001726"
+SRCREV = "b0dfd09e8d7bab8e154d2778a8131ba60c082525"
```

Comment:
This is not yet merged in master.

Reply:
Yes, now it became dependent on https://github.com/foundriesio/lmp-device-register/pull/28.

Reply:
The new version of lmp-dev-reg is merged to master, now.